### ISSUE TITLE
Try to reproduce your preprocessing for OrgansMNIST

### ISSUE THREAD

**Issue by krekiehn** [MedMNIST/MedMNIST#65]
Hello,

i try to reproduce your prerpocessing of a CT from the LIST Dataset to get the same Slices you have in OrganSMNIST.

I used a process described here:
https://github.com/MedMNIST/MedMNIST/issues/36

Thankfully you provide a csv table that maps your splitting and slice index to the source dataset, here LIST.

https://drive.google.com/drive/folders/1A_99qH_c-J0p_SatwSiaP_i1CvLUOzVo
https://drive.google.com/file/d/1g-yoc5RzPwTHY9Wz2AR7eO2jq6vIg8MQ/view?usp=drive_link

I have a question to the table. how to interpret the "image_id" string, e.g. "train-112-8-sagittal-028"?
split | index | image_id
-- | -- | --
train | 0 | train-112-8-sagittal-028

- **train in  "train-112-8-sagittal-028" means it belongs to the train split of the source dataset LIST?**
- **112 is the ID of the source CT volume?**
- **8 is the class label?**
- **sagittal the view, for sure :)**
- **what means 28? the index of the Slice in the CT? which version original spacing or uniformspacing? or index from the slice regarding the bbox?**

If 8 means the class label. it is somewhat shifted from the label classes in the code, whiche range from 0 to 10. but in the tabel it is 1 to 11.
So is the mapping 0 to 1, ...10 to 11?

    "organsmnist": {
        "python_class": "OrganSMNIST",
        "description": "The OrganSMNIST is based on 3D computed tomography (CT) images from Liver Tumor Segmentation Benchmark (LiTS). It is renamed from OrganMNIST_Sagittal (in MedMNIST v1) for simplicity. We use bounding-box annotations of 11 body organs from another study to obtain the organ labels. Hounsfield-Unit (HU) of the 3D images are transformed into gray-scale with an abdominal window. We crop 2D images from the center slices of the 3D bounding boxes in sagittal views (planes). The images are resized into 1×28×28 to perform multi-class classification of 11 body organs. 115 and 16 CT scans from the source training set are used as training and validation set, respectively. The 70 CT scans from the source test set are treated as the test set.",
        "url": "https://zenodo.org/records/10519652/files/organsmnist.npz?download=1",
        "MD5": "9ab87b696fb54e2a387ebe992d6ed5f1",
        "url_64": "https://zenodo.org/records/10519652/files/organsmnist_64.npz?download=1",
        "MD5_64": "53a6d115339d874c25e309a994ff46d3",
        "url_128": "https://zenodo.org/records/10519652/files/organsmnist_128.npz?download=1",
        "MD5_128": "ded0c5fa01a95dc4978b956f613e9b8e",
        "url_224": "https://zenodo.org/records/10519652/files/organsmnist_224.npz?download=1",
        "MD5_224": "b354719e553fbbb2513d5533f52a4cb1",
        "task": "multi-class",
        "label": {
            "0": "bladder",
            "1": "femur-left",
            "2": "femur-right",
            "3": "heart",
            "4": "kidney-left",
            "5": "kidney-right",
            "6": "liver",
            "7": "lung-left",
            "8": "lung-right",
            "9": "pancreas",
            "10": "spleen",
        },
        "n_channels": 1,
        "n_samples": {"train": 13932, "val": 2452, "test": 8827},
        "license": "CC BY 4.0",
    },

However, there seems to be a mismatch between the npz. file and the table.
The number of samples in the train split should be 13932 from the code. but from the table it is 13940. while the difference of 8 seems negligibly small, I wonder if the allocation is completely confused as a result (see below). For val it is fitting. for test it is 8827 vs. 8829.

So maybe the split table is just outdated? in that case, could you pls provide an updated version?

And if try to take a deeper look:
assuming my interpretation of the "image_id" is correct:

For "**train-000-**" there is no slice with label 7,8 and9, but if i use the indices of all slices with "train-000-" in the "Image_ID" to select the label from the .npz file all label classes are included.

From "image_id" from csv tabel: 

class  counts
    1      33
    2       9
    3      11
    4       8
    5       7
    6      12
   10      12
   11      17

Using indices from exact same cases above from the table and apply it on the npz file to the labels:

class counts
0     10
1      4
2      7
3      6
4     12
5      5
6     34
7      5
8      5
9     11
10    10

Greetings krekiehn

**Comment by krekiehn** [comment]
here are the concret list of your OrganSMINST Slice index from column "index" i used to select the corresponding label from the npz file for "**train-000-**"

[50, 476, 618, 731, 875, 969, 999, 1027, 1098, 1291, 1385, 1572, 1752, 1959, 1976, 1990, 2221, 2532, 2646, 2655, 2777, 3077, 3144, 3482, 3698, 3812, 3885, 3893, 4085, 4135, 4368, 4445, 4513, 4767, 4791, 4854, 4946, 4984, 5030, 5198, 5250, 5415, 5472, 5573, 6044, 6050, 6057, 6109, 6141, 6212, 6229, 6277, 6366, 6456, 6482, 6506, 6591, 7010, 7310, 7483, 7497, 7620, 7662, 7682, 7686, 7883, 7915, 8023, 8126, 8247, 8599, 8653, 9047, 9214, 9266, 9292, 9462, 9584, 9732, 9874, 9939, 9960, 10044, 10045, 10125, 10255, 10272, 10303, 10451, 10463, 11085, 11128, 11181, 11325, 11480, 11564, 11672, 11934, 12220, 12249, 12559, 12621, 12788, 12866, 12911, 13111, 13486, 13487, 13894]

**Comment by ThreeSRR** [comment]
Hi @krekiehn ,

Regarding the `image_id`, it follows the format {volume_filename}-{perspective}-{slice_index}, where the `volume_filename` refers to the normalized volume data processed from the original LiTS dataset. Unfortunately, we failed to find the scripts to generate these files :(

For your convenience, we have uploaded the processed volumes [here](https://drive.google.com/file/d/1W5vTj9myAlEQ4UUnxFjBg3KRNGHHBSYq/view?usp=sharing).
